src/api/docs/doc.test.js

```diff
@@ -17,15 +17,41 @@ afterAll(async () => {
 });
 
 describe('GET /apiDocs/', () => {
-  it('Should display "Available versions" text on page', async () => {
+  it('Should display "Available versions" text on page.', async () => {
     await page.goto(`${BASE_URL}/apiDocs/`);
     await expect(page).toMatch('Available versions');
   });
 });
 
 describe('GET /apiDocs/v1.0.0', () => {
-  it('Should display "CodingGarden Community App APIs" text on page', async () => {
+  it('Should display "CodingGarden Community App APIs" text on page.', async () => {
     await page.goto(`${BASE_URL}/apiDocs/v1.0.0`);
     await expect(page).toMatch('CodingGarden Community App APIs');
   });
 });
+
+describe('GET API response through API Doc', () => {
+  it('Should be able to query /contributors API from Doc page.', async () => {
+    await page.goto(`${BASE_URL}/apiDocs/`);
+
+    await page.waitForSelector('body > ul > li > a');
+    await page.click('body > ul > li > a');
+
+    try {
+      await page.waitForSelector('div > span > #operations-Contributors-get_contributors > .opblock-summary > .opblock-summary-description');
+      await page.click('div > span > #operations-Contributors-get_contributors > .opblock-summary > .opblock-summary-description');
+
+      await page.waitForSelector('.opblock-body > .opblock-section > .opblock-section-header > .try-out > .btn');
+      await page.click('.opblock-body > .opblock-section > .opblock-section-header > .try-out > .btn');
+
+      await page.waitForSelector('#operations-Contributors-get_contributors > div > .opblock-body > .execute-wrapper > .btn');
+      await page.click('#operations-Contributors-get_contributors > div > .opblock-body > .execute-wrapper > .btn');
+
+      await page.waitForSelector('.response > .col > div > .highlight-code > .microlight');
```

Comment:
Yes, not directly. But this element only exists after a successful request. By that, we can conclude the request was successful.

Do you have any suggestion to directly check the request's success? I 'll look into this more in the meantime.

Reply:
Sorry, I meant "successful request" in my original comment. 

Might not be necessary to check it because we already have tests making sure the routes works, but we can go the extra mile and check for status code in the response body. What do you think.

Reply:
This check is not about whether APIs work or not. It's whether the API doc can call them or not through UI. That's why I thought it's necessary. It's the part where end-to-end testing comes in, I suppose(?)

The second part actually makes a lot of sense. It is an accurate thing to do. That's what I originally considered doing. But I am at the limit of my knowledge about Puppeteer. I've been looking through their documentation. I'll update it with actual response code check. That's a better thing to do.

Reply:
Sounds good, we should make sure the CI tests pass before adding anything else. Do you have any ideas why the tests are failing?

Reply:
Probably because of the `const BASE_URL = `http://${HOST}:${PORT}`;`
Depends on CI's env HOST variable. If it's anything other than `localhost`, that's definitely the reason. And changing my code back to const BASE_URL = `http://localhost:${PORT}`; will fix that.

If that's not the case, I am not sure. We will get more info once I push with above-mensioned change. 
Let me actually do that right now.

Reply:
I was thinking that the localtest URL is hard-coded on port 3000, and the travis ci test can possibly run on other ports. Try changing the host back to "localhost" and see if that fixes anything.

Reply:
Yup, that did it. :)